internal/provider/alert_resource_acc_test.go

```diff
@@ -104,18 +105,17 @@ func TestAccMetricGroupAlertResource(t *testing.T) {
 					resource.TestCheckResourceAttr("swo_alert.test", "conditions.0.exclude_tags.0.name", "service.name"),
 					resource.TestCheckResourceAttr("swo_alert.test", "conditions.0.exclude_tags.0.values.0", "test-service"),
 
-					resource.TestCheckResourceAttr("swo_alert.test", "notifications.0", "123"),
-					resource.TestCheckResourceAttr("swo_alert.test", "notifications.1", "456"),
 					resource.TestCheckResourceAttr("swo_alert.test", "runbook_link", "https://www.runbooklink.com"),
 					resource.TestCheckResourceAttr("swo_alert.test", "trigger_delay_seconds", "300"),
 				),
 			},
 			// ImportState testing
-			/*{
-				ResourceName:      "swo_alert.test",
-				ImportState:       true,
-				ImportStateVerify: true,
-			}*/
+			{
+				ResourceName:            "swo_alert.test",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"notification_actions"},
```

Comment:
This exception is due to the fact that the mapping for actions is not reversible, due to the very nature of the model that was chosen. Therefore, we might end up representing the same notifications in a way that is slightly different than it was specified in the config, which would make this particular attribute fail the check.